[Area 1] Inline setup markdown migration (#6074)
cloudflare/cloudflare-docs: content/email-security/deployment/inline/setup/office-365-area1-mx.md

---
title: Office 365 - Area 1 as MX Record
pcx_content_type: tutorial
weight: 1
layout: single
meta:
   title: Deploy and configure Microsoft Office 365 with Area 1 as MX Record
---

# Deploy and configure Microsoft Office 365 with Area 1 as MX Record

![A schematic showing where Area 1 security is in the life cycle of an email received](/email-security/static/office365-mx.png)

> In this tutorial you will learn how to configure Microsoft Office 365 with Area 1 as MX record. This tutorial is broken down into several steps.

Reviewer: ```suggestion
In this tutorial, you will learn how to configure Microsoft Office 365 with Area 1 as MX record. This tutorial is broken down into several steps.
```

    ```txt
    PS C:\Windows\system32> Connect-ExchangeOnline
    ```

3. The `Connect-ExchangeOnline` cmdlet will prompt you to login. Log in using an Office 365 administrator account. Once authenticated, you will be returned to the PowerShell prompt.

4. You can verify that the `OrganizationCustomization` is enabled by running the command:

```txt
PS C:\Windows\system32> Get-OrganizationConfig | FL isDehydrated
```

If the result is `false`, `OrganizationCustomization` is already enabled and no further actions are required. If it is true, you need to enable it:

```txt
PS C:\> Enable-OrganizationCustomization
```

## 3. Enhanced Filtering configuration

This option will allow Office 365 to properly identify the original connecting IP before the message was received by Area 1. This helps with SPF analysis. This has two steps: 

* Creating an inbound connector.
* Enabling the enhanced filtering configuration of the connector.

### Create an inbound connector

1. From the **Microsoft Exchange admin center**, select **mail flow** > **connectors**.

2. Select the `+` icon to configure a new connector. In the **Select your mail flow scenario panel** dialog box that opens:

>     1. In the *From* drop-down: select **Partner organization**.

Reviewer: ```suggestion
    1. In the *From* dropdown: select **Partner organization**.
```

>     2. In the *To* drop-down: select **Office 365**.

Reviewer: ```suggestion
    2. In the *To* dropdown: select **Office 365**.
```

{{<Aside type="note">}}When Area 1 is deployed as the MX record and protecting Office 365, Malicious and Spam detections will automatically be quarantined. This behavior cannot be modified.{{</Aside>}}

### Manage the Admin Quarantine

1. Log in to the [Area 1 dashboard](https://horizon.area1security.com/).

2. Go to **Email** > **Admin Quarantine**.

3. Locate the message you want to manage, and select the `...` icon next to it. Thill will let you preview, download, or release the quarantined message.

### Optional - Quarantine messages using Microsoft Hosted Quarantine

As previously noted, malicious and spam detections are automatically quarantined in Area 1’s quarantine (this behavior cannot be modified). However, for the suspicious and spoof dispositions, you may prefer to apply a different behavior, where these messages can be quarantined into the Microsoft Hosted Quarantine or sent to the user’s junk folder.

For this alternate behavior, you will need to configure a transport rule in Office 365:

1. Go to the **Exchange administrator** console > **mail flow** > **rules**.

2. Select the **+** button > **Create a new rule**.

> 3. In the new dialog box, select **More options…** to open the advanced version of the rule creator. Set the following conditions and actions:

Reviewer: ```suggestion
3. In the new dialog box, select **More options** to open the advanced version of the rule creator. Set the following conditions and actions:
```

Reviewer: I think we decided that we would include ellipses: https://wiki.cfops.it/pages/viewpage.action?pageId=254838248

* **Name**: `Quarantine Area 1 Suspicious Messages`

> * Configure the first condition, select **A message header…** > **Includes any of these words:**

Reviewer: ```suggestion
* Configure the first condition, select **A message header** > **Includes any of these words:**
```

    * Enter text: `X-Area1Security-Disposition`
    * Enter words: `SUSPICIOUS`

{{<Aside type="note">}}If you also want to quarantine the spoof detections, add the string `SPOOF` to the list of words.{{</Aside>}}

4. Select the **add** condition button to add a second condition.

5. In the new condition, select **The sender** > **IP address is in any of these ranges or exactly matches** and enter the egress IPs in the [Egress IPs page](/email-security/deployment/inline/reference/egress-ips/).

> 6. In the **Do the following…** section, select **Redirect the message to**  > **hosted quarantine.**.

Reviewer: ```suggestion
6. In the **Do the following** section, select **Redirect the message to**  > **hosted quarantine.**.
```

{{<Aside type="note">}}
If you prefer to send the message to the Junk folder, in the **Do the following** section, select **Modify the message properties**  > **set the spam confidence level (SCL)**. Then, select the SCL value that will send the message to the junk folder. This behavior is dependent on the configured spam filter policies (spam and bulk actions).
{{</Aside>}}

7. Select **Save** to save the new rule.

## 5. Update your domain MX records

Instructions to update your MX records will depend on the DNS provider you are using. You will need to update and replace your existing MX record with the Area 1 hosts:

> MX Priority | Host

Reviewer: Same here.
